src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -251,6 +317,21 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import com.neurologic.http.impl.ApacheHttpClient; // OK
+ * import static java.awt.Button.A;
+ * import javax.swing.JComponent; // violation, wrong order caused by above static import
+ *                      // all static imports comes at bottom
+ * import java.net.URL; // violation, extra separation  in import group before 'java.net.URL'
```

Comment:
same here, no need to copy import class into comment

Reply:
done